election/views.py

```diff
@@ -1,4 +1,9 @@
+import numpy as np
+from django.db.models import Count
 from django.db import IntegrityError
+from django.core.exceptions import EmptyResultSet
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
```

Comment:
impoted but not used. It seems to me that we had said the server was language agnostic and that all the translation work was done in the front end.